subprojects/packagefiles/quill/meson.build

```diff
@@ -0,0 +1,20 @@
+project('quill', 'cpp', version : '4.2.0', default_options : ['warning_level=3', 'cpp_std=c++17'])
+
+inc_dirs = include_directories('quill/include')
+
+# Conditional compiler arguments for specific compiler
+interface_compile_args = meson.get_compiler('cpp').get_supported_arguments('-Wno-gnu-zero-variadic-macro-arguments')
+
+quill_dep = declare_dependency(include_directories : inc_dirs,
+                               dependencies : [dependency('threads')],
+                               compile_args : interface_compile_args)
+
+install_subdir('quill/include/quill', install_dir: get_option('includedir') / 'quill', install_tag: 'devel', strip_directory: true)
```

Comment:
A simpler version of this should be

    install_subdir('quill/include/quill', install_dir: get_option('includedir')', install_tag: 'devel')